examples/conversationbot.py

```diff
@@ -23,7 +23,7 @@
 except ImportError:
     __version_info__ = (0, 0, 0, 0, 0)  # type: ignore[assignment]
 
-if __version_info__ < (20, 0, 0, "alpha", 1):
+if __version_info__ < (20, 0, 0, "alpha", 5):
```

Comment:
Maybe in the release page? But it's hard to say when a change breaks an example post those changes

Reply:
I'm mainly talking about our plans to make the next release a beta release. Only if we get another API update before #3249 is mergable, the next release would be calld 20.0a5 … Mind you, this check would still be correct, since `(20, 0, 0, "alpha", 5) < (20, 0, 0, "beta", 0)`, but it would probably still confuse users xD I'll add a note o the "how to release page" as reminder :)